warehouse/macaroons/models.py

```diff
@@ -52,10 +52,9 @@ class Macaroon(db.Model):
     created = Column(DateTime, nullable=False, server_default=sql.func.now())
     last_used = Column(DateTime, nullable=True)
 
-    # We'll store the caveats that were added to the Macaroon during generation
-    # to allow users to see in their management UI what the total possible
-    # scope of their macaroon is.
-    caveats = Column(JSONB, nullable=False, server_default=sql.text("'{}'"))
+    # Human-readable "permissions" for this macaroon, corresponding to the
+    # body of the permissions ("V1") caveat.
+    permissions = Column(JSONB, nullable=False, server_default=sql.text("'{}'"))
```

Comment:
```suggestion
    permissions_caveat = Column(JSONB, nullable=False, server_default=sql.text("'{}'"))
```

I think one thing that's throwing me off here is that (IIUC) this is actually a caveat, not the permissions. So later when we're doing `macaroon.permissions.permissions` this could be `macaroon.permissions_caveat.permissions` and be a lot more clear which we're referring to elsewhere.

Reply:
Yep. I'll rename everywhere.

Reply:
[3e4c601](https://github.com/pypa/warehouse/pull/11157/commits/3e4c601ffd5fca9176243d418afe00dd8ab5a2aa)